CHANGELOG.md

```diff
@@ -10,11 +10,17 @@
 
 ### Removed
 
+## v1.3.0
+
+### Added
+
+- [PR #278](https://github.com/konpyutaika/nifikop/pull/278) - **[Operator]** Added the single-user-authentication method.
+
```

Comment:
```suggestion
```suggestion
## Unreleased

### Added
- [PR #278](https://github.com/konpyutaika/nifikop/pull/278) - **[Operator]** Added the single-user-authentication method.

### Changed

### Fixed Bugs

### Deprecated

### Removed

```
```

Reply:
Modified accordingly